src/main/java/com/easypost/model/CreditCard.java

```diff
@@ -74,11 +70,16 @@ public static void delete(String creditCardId) throws EasyPostException {
      * Delete a credit card by ID.
      *
      * @param creditCardId ID of credit card to delete.
-     * @param apiKey API key to use in request (overrides default API key).
+     * @param apiKey       API key to use in request (overrides default API key).
      * @throws EasyPostException when the request fails.
      */
     public static void delete(String creditCardId, String apiKey) throws EasyPostException {
-        request(RequestMethod.DELETE, String.format("%s/%s/%s", EasyPost.API_BASE, "credit_cards",
-            creditCardId), null, CreditCard.class, apiKey);
+        request(RequestMethod.DELETE, String.format("%s/%s/%s", EasyPost.API_BASE, "credit_cards", creditCardId), null,
+                CreditCard.class, apiKey);
+    }
+
+    public enum Priority {
```

Comment:
I'd expect enums to be at a top of a file, not the very bottom.

Reply:
Sounds good, will bump.